src/nwm_routing/src/nwm_routing/preprocess.py

```diff
@@ -190,17 +193,13 @@ def nwm_initial_warmstate_preprocess(
         print("setting channel initial states ...")
 
     q0 = nnu.build_channel_initial_state(restart_parameters, segment_index)
-
+    
     if verbose:
         print("channel initial states complete")
     if showtiming:
         print("... in %s seconds." % (time.time() - start_time))
         start_time = time.time()
 
-    # TODO: Does this need to live outside the if-block for waterbodies above? If not, let's move it up there to keep things together.
-    waterbodies_df = pd.merge(
-        waterbodies_df, waterbodies_initial_states_df, on="lake_id"
-    )
 
```

Comment:
- [ ] need to check whether this is relevant to the PR -- if not, we can wrap it in a quick separate PR.